### PR TITLE
User Guide - Formatting and updating/removal of certain sections

### PR DESCRIPTION
**Pull request by HM33-Stu**
UG formatting and updating/removal of certain sections

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,11 +3,18 @@ layout: page
...
-  {:toc}
+### Table Of Contents
+
+  | Section                             | When Should I Use This?                                                 |
+  |-------------------------------------|-------------------------------------------------------------------------|
+  | [Quick Start](#quick-start)         | If you are using the application for the first time or need a refresher |
+  | [Features](#features)               | If you would like to know the capabilities of the application           |
+  | [FAQ](#faq)                         | If you have any questions about the application                         |
+  | [Known Issues](#known-issues)       | If you have a technical problem. We may have the solution!              |
+  | [Command Summary](#command-summary) | If you would like to find more information about a specific command     |
```
Comment (HM33-Stu) [inline docs/UserGuide.md:17]: Added in table of contents

File: docs/UserGuide.md [hunk 2]
```diff
@@ -17,19 +24,18 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
...
-    - [How do I check my version of Java installed?](#check-version-of-java-installed)
     - If you are a MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html) instead.
-2. Download the latest `CareerSync.jar` file [here]().
+2. Download the latest `CareerSync.jar` file [here](https://github.com/AY2324S2-CS2103T-W11-1/tp/releases/tag/v1.2).
 3. Make sure your jar file is located in an empty folder.
 4. Start <span style="color: #f66a0a;">CareerSync</span>.
     - For MacOS:
         - Open up your Terminal by typing <kbd>Command</kbd> + <kbd>Space</kbd>, then type <kbd>Enter</kbd>.<br>
-        - Navigate to the folder containing your jar file using `cd`. If you are not sure how to use `cd`, refer to [this link]!(https://www.ibm.com/docs/en/aix/7.2?topic=directories-changing-another-directory-cd-command).<br>
+        - Navigate to the folder containing your jar file using `cd`. If you are not sure how to use `cd`, refer to [this link](https://www.ibm.com/docs/en/aix/7.2?topic=directories-changing-another-directory-cd-command)!<br>
```
Comment (HM33-Stu) [inline docs/UserGuide.md:33]: Link was originally entered as text, rectified the issue

File: docs/UserGuide.md [hunk 3]
```diff
@@ -17,19 +24,18 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
 ### Installation
 
 1. Ensure that you have [Java 11 or above](https://www.java.com/en/download/) installed on your computer.
-    - [How do I check my version of Java installed?](#check-version-of-java-installed)
     - If you are a MacOS, follow the instructions [here](https://nus-cs2103-ay2324s2.github.io/website/admin/programmingLanguages.html) instead.
-2. Download the latest `CareerSync.jar` file [here]().
+2. Download the latest `CareerSync.jar` file [here](https://github.com/AY2324S2-CS2103T-W11-1/tp/releases/tag/v1.2).
```
Comment (HM33-Stu) [inline docs/UserGuide.md:28]: Added in link to our v1.2 release

File: docs/UserGuide.md [hunk 4]
```diff
@@ -56,16 +62,13 @@ Start keeping track of your internships easily with <span style="color: #f66a0a;
...
+**Notes about the command format**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add /com COMPANY_NAME`, `COMPANY_NAME` is a parameter which can be used as `add /com Google`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
```
Comment (HM33-Stu) [inline docs/UserGuide.md:68]: Removal of TAG section

File: docs/UserGuide.md [hunk 5]
```diff
@@ -129,7 +131,7 @@ CLI input will be implemented in a future release.
 
 Edits an existing internship entry in the application.
 
-Format: `edit INDEX [/company COMPANY_NAME] [/pocname NAME_OF_CONTACT] [/email EMAIL_OF_CONTACT] [/number NUMBER_OF_CONTACT] [/location LOCATION_ENUM] [/status STATUS] [/description DESCRIPTION] [/role ROLE] …​`
+Format: `edit INDEX [/com COMPANY_NAME] [/poc CONTACT_NAME] [/email CONTACT_EMAIL] [/phone CONTACT_NUMBER] [/loc LOCATION_ENUM] [/status STATUS] [/desc DESCRIPTION] [/role ROLE] …​`
```
Comment (HM33-Stu) [inline docs/UserGuide.md:134]: Renamed to match with the other sections

File: docs/UserGuide.md [hunk 6]
```diff
@@ -211,19 +211,20 @@ _Details coming soon ..._
...
-| [exit](#exiting-the-program-exit)          | Exits and closes the application.        |
+| Action                                         | Description                              |
+|------------------------------------------------|------------------------------------------|
+| [add](#adding-an-entry-add)                    | Adds an Internship.                      |
+| [delete](#deleting-an-internship-delete)       | Removes a Internship.                    |
+| [list](#listing-all-internships-list)          | Removes a Internship.                    |
+| [edit](#editing-an-internship-edit)            | Modifies an existing Internship.         |
+| [addremark](#adding-a-remark-addremark)        | Adds a remark to an existing Internship. |
+| [clear](#clearing-all-internships-clear)       | Removes all Internships from the deck.   |
+| [find](#locating-internships-by-keywords-find) | Sets the goal for the session.           |
```
Comment (HM33-Stu) [inline docs/UserGuide.md:229]: Rectified find link

**Review by jieqiboh — APPROVED** [review]

**Pull request merged**